public/content/translations/zh-tw/developers/docs/blocks/index.md

```diff
@@ -39,97 +39,97 @@ lang: zh-tw
+| `voluntary_exits`    | 離開網路的驗證者清單     |
+| `sync_aggregate`     | 服務輕量用端的驗證者子集   |
+| `execution_payload`  | 執行用户端傳送來的交易    |
 
 `attestations` 欄位包含區塊中所有證明的清單。 每個證明都有自己的資料類型並包含一些資料。 每個證明包含：
 
-| 欄位               | 描述                     |
-| :----------------- | :----------------------- |
+| 欄位                 | 描述           |
+|:------------------ |:------------ |
 | `aggregation_bits` | 參與過此證明的驗證者清單 |
-| `數據資料`         | 包含多個子欄位的容器     |
+| `數據資料`             | 包含多個子欄位的容器   |
 | `signature`        | 所有證明驗證者的聚合簽名 |
 
 `attestation` 中的 `data` 欄位包含：
 
-| 欄位                | 描述                           |
-| :------------------ | :----------------------------- |
-| `時隙`              | 與證明相關的時隙               |
-| `索引`              | 證明驗證者的索引               |
+| 欄位                  | 描述              |
+|:------------------- |:--------------- |
+| `時隙`                | 與證明相關的時隙        |
+| `索引`                | 證明驗證者的索引        |
 | `beacon_block_root` | 包含此物件的信標區塊的根雜湊值 |
-| `來源`              | 最後一個合法檢查點             |
-| `target`            | 最新時期的邊界區塊             |
+| `來源`                | 最後一個合法檢查點       |
+| `target`            | 最新時期的邊界區塊       |
 
 執行 `execution_payload` 中的交易會更新全域狀態。 所有用戶端都重新執行 `execution_payload` 中的交易，以確保新的狀態與新區塊中 `state_root` 欄位中的狀態相符。 這就是用戶端辨別新區塊是否有效並可以安全添加至其區塊鏈中的方式。 `execution payload` 自身是一個有許多欄位的物件。 還有一個 `execution_payload_header` 欄位，其中包含了關於執行資料的重要摘要資訊。 這些資料結構組織方式如下：
 
 `xecution_payload_header` 包含以下欄位：
 
-| 欄位                | 描述                                     |
-| :------------------ | :--------------------------------------- |
-| `家長_雜湊值`       | 父區塊的雜湊值                           |
-| `fee_recipient`     | 用於支付交易費的帳戶地址                 |
+| 欄位                  | 描述                   |
+|:------------------- |:-------------------- |
+| `家長_雜湊值`            | 父區塊的雜湊值              |
+| `fee_recipient`     | 用於支付交易費的帳戶地址         |
 | `state_root`        | 在應用此區塊中的變更後全域狀態的根雜湊值 |
-| `receipts_root`     | 交易收據樹的雜湊值                       |
-| `logs_bloom`        | 包含事件記錄的資料結構                   |
-| `prev_randao`       | 用於隨機選擇驗證者的值                   |
-| `block_number`      | 目前區塊號碼                             |
-| `gas_limit`         | 此區塊允許的最高燃料用量                 |
-| `gas_used`          | 此區塊實際消耗的燃料用量                 |
-| `時間戳`            | 區塊時間                                 |
-| `extra_data`        | 原始字節位元組格式的任意額外資料         |
-| `base_fee_per_gas`  | 基本費用的值                             |
-| `block_hash`        | 執行區塊的雜湊值                         |
-| `transactions_root` | 有效負載中交易的根雜湊值                 |
-| `withdrawal_root`   | 有效負載中提款的根雜湊值                 |
+| `receipts_root`     | 交易收據樹的雜湊值            |
+| `logs_bloom`        | 包含事件記錄的資料結構          |
+| `prev_randao`       | 用於隨機選擇驗證者的值          |
+| `block_number`      | 目前區塊號碼               |
+| `gas_limit`         | 此區塊允許的最高燃料用量         |
+| `gas_used`          | 此區塊實際消耗的燃料用量         |
+| `時間戳`               | 區塊時間                 |
```

Comment:
Hm, doesn't look like this is new, but this is the `timestamp` property... Given these are property names, I'm not sure we should be translating these (similar to the others). 

Will leave it as is for this PR, but flagging to discuss in the future. cc: @lukassim

Reply:
...or if we're going to translate them, we should translate them all